test/QtDBMigrationTest.cpp

```diff
@@ -88,11 +88,15 @@ void QtDBMigrationTest::migrate_data()
     QTest::addColumn<QString>("configPath");
     QTest::addColumn<QStringList>("tableNames");
     QTest::addColumn<int>("finalVer");
-
+    QString folderPath = qEnvironmentVariable("MIGRATIONS_PATH");
     QTest::newRow("normal")
         << "test_migrate_1.json"
         << (QStringList() << "table1" << "table2")
         << 1;
+    QTest::newRow("Migrate folder: init -> 0")
```

Comment:
Have you also tested if migration folder is not existant?

Try to cover all edge cases (look at the `if`s and other loops in the code). Main point of tests is to cover all the lines, if possible.

Reply:
Have you forgot to commit the test migrations?

Reply:
Added more tests. Also added the test migrations.

Reply:
:+1: